components/export/ui/src/main/resources/PhenoTips/ExportPreferences.xml

```diff
@@ -93,27 +157,21 @@
 {{/velocity}}
 
 {{velocity}}
-{{html clean="false" filter="indent"}}&lt;form class="xform" action="${escapetool.xml($request.get('export_endpoint'))}" method="post"&gt;&lt;div&gt;{{/html}}
-(% class="section filters"%)(((
-=== (% class="step" %)1(%%) $services.localization.render('phenotips.exportPreferences.filterTitle') ===
-
-#__filters_display('export-filters')
-
-{{info}} $services.localization.render('phenotips.exportPreferences.numMatchingRecords', [ "**(% id='filter-match-count'%) (%%)**" ]) {{/info}}
-)))##section filters
-
-(% class="section columns"%)(((
-=== (% class="step" %)2(%%) $services.localization.render('phenotips.exportPreferences.selectFields') ===
-#if ($request.multipatient)
-  #set($activeFields = ['external_id', 'gender', 'phenotype', 'negative_phenotype', 'omim_id'])
-  #set($ignored = {})
-  #set($customValues = {})
+#if ($request.push)
+  #set($customValues = {
+       'allergies' : "$services.localization.render('PhenoTips.AllergiesDataClass_allergies')",
+       'genes': "$services.localization.render('phenotips.exportPreferences.field.genes')",
+       'genes_comments': "$services.localization.render('phenotips.exportPreferences.field.genesComments')",
+       'rejectedGenes': "$services.localization.render('phenotips.exportPreferences.field.rejectedGenes')",
+       'rejectedGenes_comments': "$services.localization.render('phenotips.exportPreferences.field.rejectGenesComments')"
```

Comment:
1) yes, if this code is used
2) this code is actually no longer used after Marta's changes (https://github.com/phenotips/phenotips/commit/f7581aa1f7c8cd299382204759fb009cc804b327), so should be removed